app/routes/calendar/new.tsx

```diff
@@ -287,14 +287,42 @@ function DescriptionTextarea() {
   );
 }
 
+/**
+ * Retrieves a new Date object that is offset by several hours.
+ * 
+ * NOTE: it is important that we work with & return a copy of the date here, 
+ *  otherwise we will just be mutating the original date passed into this function.
+ * 
+ * @param date 
+ * @returns 
```

Comment:
Imo these are not needed when using TS.

Reply:
Oh yeah sure, that is just what the `/** <press space>` macro does in VSCode (automatically makes this Javadoc-like header comment for the function). I have removed it as requested